indexing-service/src/main/java/io/druid/indexing/common/stats/RowIngestionMeters.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.indexing.common.stats;
+
+import java.util.Map;
+
+/**
+ * A collection of meters for row ingestion stats, with support for moving average calculations.
+ * This can eventually replace FireDepartmentMetrics, but moving averages for other stats collected by
+ * FireDepartmentMetrics are not currently supported, so we continue to use FireDepartmentMetrics alongside
+ * RowIngestionMeters to avoid unnecessary overhead from maintaining these moving averages.
+ */
+public interface RowIngestionMeters
```

Comment:
Would you explain why this is an interface? Is it for smoothly migrating to some other libraries instead of Dropwizard in the future?

Reply:
It's an interface so that extensions can provide custom implementations, see https://github.com/druid-io/druid/pull/5748#discussion_r187757058

Reply:
Thanks.

Reply:
Does it make sense to make this as an `ExtensionPoint`?

Reply:
Added @ExtensionPoint